### ISSUE TITLE
[Heat Log] Separate heat logs by user

### ISSUE THREAD

**Issue by williamcodes** [clhenrick/help-i-need-heat#14]
To be used by more than one person, users need to have their own individual heat logs. Tricky to implement though. You'll need some kind of authentication mechanism. Also, using a single cartodb table won't cut it anymore.

**Comment by clhenrick** [comment]
Right, the heat log as is was just a proof of concept for the hack event. In reality this part of the site needs to be more complex. Do we need to have users create an account? Or perhaps just sign in with social media?

**Comment by rstachel** [comment]
FYI - you don't need separate tables to support users. In fact, there is a
User column in the existing table (though not yet used). The database API
supports selecting only the table rows that belong to a particular user.

So, given that the app authenticates by user, it is not hard to support
per-user log data.

rstachel@gmail.com
(781) 333-UBOB [8262]

**Comment by clhenrick** [comment]
thanks @rstachel for the heads up!
